slack-server-integration-common/src/main/java/com/atlassian/plugins/slack/rest/SlackSignatureVerifyingFilter.java

```diff
@@ -159,6 +159,7 @@ private void validateSignature(final String teamId,
                             .toString();
                     String expectedSignature = "v0=" + encodeHmacSha256(signingSecret, payloadToHash);
                     if (!Objects.equals(expectedSignature, actualSignature)) {
+                        LOG.error("Request signature verification failed. Expected signature: {} and payload: {}", expectedSignature, payloadToHash);
```

Comment:
It is exactly the right place for this log!
Could you also add `actualSignature` to the log message to make it possible to potentially provide this information to Slack support in case of need?

Reply:
Added